pypesto/objective/jax/base.py

```diff
@@ -42,7 +54,9 @@ def _device_fun(base_objective: ObjectiveBase, x: jnp.array):
         jax computed input array.
     """
     return jax.pure_callback(
```

Comment:
Short documentation on the return structure would be helpful.

Reply:
added